### PR TITLE
add(route): add hdu cs postgraduate notice route and pku ss route

### PR DESCRIPTION
**Pull request by legr4ndk**
## 该 PR 相关 Issue / Involved issue

无相关issue
Close #

## 完整路由地址 / Example for the proposed route(s)

```routes

/hdu/cs/pg
/pku/ss/notice
/pku/ss/admission
/pku/ss/pgadmin
```

## 新 RSS 检查列表 / New RSS Script Checklist
  
- [ + ] 新的路由 New Route
  - [ + ] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [ + ] 文档说明 Documentation
  - [ + ] 中文文档 CN
  - [ ] 英文文档 EN
- [ + ] 全文获取 fulltext
  - [ + ] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [ + ] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [ + ] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

## 说明 / Note

增加了一个杭电的router与三个软微的router

### CODE REVIEW

**Comment by legr4ndk** [comment]
;( i used npm install and my yarn.lock files cant pass the check
idk why
in the second check i cant even pass the _Validate PR title_ check

**Comment by legr4ndk** [comment]
maybe i will have to rewrite all of this and use yarn instead of npm?

File: yarn.lock [hunk 1]
```diff
@@ -4,27 +4,27 @@
 
 "@ampproject/remapping@^2.1.0":
   version "2.2.0"
-  resolved "https://registry.yarnpkg.com/@ampproject/remapping/-/remapping-2.2.0.tgz#56c133824780de3174aed5ab6834f3026790154d"
+  resolved "https://registry.npmmirror.com/@ampproject/remapping/-/remapping-2.2.0.tgz"
```
Comment (TonyRL) [inline yarn.lock:7]: Please remove any changes on yarn.lock unless you're adding a new package

File: lib/v2/hdu/cs/pg.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+const { parseDate } = require('@/utils/parse-date');
+
+const link = 'https://computer.hdu.edu.cn';
+const host = 'https://computer.hdu.edu.cn/6769/list.htm';
+
+const getSingleRecord = async () => {
+    const res = await got({
+        method: 'get',
+        url: host,
+    });
```
Comment (TonyRL) [inline lib/v2/hdu/cs/pg.js:12]:
```suggestion
    const res = await got(host);
```

File: lib/v2/hdu/radar.js [hunk 3]
```diff
@@ -1,13 +1,19 @@
 module.exports = {
-    'hdu.edu.cn': {
-        _name: '杭州电子科技大学',
-        computer: [
+    'computer.hdu.edu.cn': {
+        _name: '杭州电子科技大学-计算机学院',
+        '.': [
```
Comment (TonyRL) [inline lib/v2/hdu/radar.js:4]:
```suggestion
    'hdu.edu.cn': {
        _name: '杭州电子科技大学',
        'computer': [
```

**Comment by legr4ndk** [comment]
ok i will close this pr and fix yarn.lock and make a new one
thank u tony

**Pull request closed without merging**